library/db/src/main/kotlin/org/cru/godtools/db/repository/LanguagesRepository.kt

```diff
@@ -0,0 +1,21 @@
+package org.cru.godtools.db.repository
+
+import java.util.Locale
+import kotlinx.coroutines.flow.Flow
+import org.cru.godtools.model.Language
+
+interface LanguagesRepository {
```

Comment:
the common `LanguagesRepository` interface that doesn't have any implementation logic in it